lib/smile-identity-core/web_api.rb

```diff
@@ -108,6 +109,10 @@ def options=(options)
 
     private
 
+    def symbolize_keys hash
+      (hash.is_a?(Hash)) ? Hash[hash.map{ |k, v| [k.to_sym, v] }] : hash
```

Comment:
This is going to break if someone inputs nil for one of the values. We should handle that case. For instance, for a job without id, nil would seem a sane input for the id_info value.